### PR TITLE
LICENSE CC-BY-SA added

### PR DESCRIPTION
**Pull request by keachi**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,9 +2,12 @@
 ANSIBLE GUIDE
 =============
 
-This repository provides an example project with the Ansible roles of
-Adfinis SyGroup AG. In additional, it contains a guide to Ansible, written
-with `Sphinx Documentation`_.
+.. image:: https://img.shields.io/github/license/adfinis-sygroup/ansible-guide.svg?style=flat-square
+  :target: https://github.com/adfinis-sygroup/ansible-guide/blob/master/LICENSE
+
+This repository provides an example project with the ansible roles written by
```
Comment (karras) [inline README.rst]: s/ansible/Ansible/
s/with/for/

File: README.rst [hunk 2]
```diff
@@ -2,9 +2,12 @@
...
+Adfinis SyGroup AG. In addition, it contains a guide to ansible, written
```
Comment (karras) [inline README.rst]: s/ansible/Ansible/

File: README.rst [hunk 3]
```diff
@@ -2,9 +2,12 @@
...
+with Sphinx-doc_.
```
Comment (karras) [inline README.rst]: Fix this link please.

**Review by karras — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (keachi) [reply README.rst]: done

**Review by karras — APPROVED** [review]

**Pull request merged**